deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/cntk/ImageFeaturizer.scala

```diff
@@ -3,22 +3,26 @@
 
 package com.microsoft.azure.synapse.ml.cntk
```

Comment:
move it out to a different package?

Reply:
I will do some package moves in a subsequent PR. It makes it hard to review if whole files are moved to diff path (and git doesn't always figure it out properly).  It's better to see the diffs.  There is a TODO in one file to move packages later.